Makefile

```diff
@@ -134,6 +134,10 @@ binary: chaosdaemon manager chaosfs chaos-dashboard bin/pause bin/suicide
 watchmaker:
 	$(CGOENV) go build -ldflags '$(LDFLAGS)' -o bin/watchmaker ./cmd/watchmaker/...
 
+# Build chaosctl
+chaosctl:
+	go install ./cmd/chaosctl
```

Comment:
`$(GO) build -ldflags '$(LDFLAGS)' -o bin/chaosctl ./cmd/chaosctl/*.go`, generate the executable files in `bin` folder.

Reply:
BTW, @Yiyiyimu please remove the binary file `pkg/chaosctl/chaosctl `.

Reply:
thx! fixed